eth/tracers/internal/tracers/prestate_tracer.js

```diff
@@ -55,7 +55,7 @@
 		var toBal   = bigInt(this.prestate[toHex(ctx.to)].balance.slice(2), 16);
 
 		this.prestate[toHex(ctx.to)].balance   = '0x'+toBal.subtract(ctx.value).toString(16);
-		this.prestate[toHex(ctx.from)].balance = '0x'+fromBal.add(ctx.value).toString(16);
+		this.prestate[toHex(ctx.from)].balance = '0x'+fromBal.add(ctx.value).add(21000 * ctx.gasPrice).toString(16);
```

Comment:
Seems like you went through a lot of trouble to get the gasprice, but you just hardcode `21K` for gas? Would better to use the actual value, somehow